cypress/e2e/app.spec.cy.ts

```diff
@@ -0,0 +1,44 @@
+let credentials;
+
+describe('application serving', () => {
+  before(() => {
```

Comment:
Should be `beforeEach` and cleanup should take place in `afterEach`. Even though this can be intensive, but it makes sure every test has clean slate to work on.

Reply:
Made the changes already, but I'm thinking that scenarios should be separate from the general tests.